test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -48,7 +48,60 @@ Feature: Bind a single application to multiple services
                     kind: Database
                     resourceRef: db-demo-2
             """
-        Then "nodejs-app" deployment must contain SBR name "binding-request-1" and "binding-request-2"
+        Then deployment must contain intermediate secret
+            | intermediate_secret_name |
+            | binding-request-1        |
+            | binding-request-2        |
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-1" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-1" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding Request "binding-request-2" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-2" should be changed to "True"
+
+
+    Scenario: Bind two backend services by creating 2 SBRs to a single application
```

Comment:
This scenario is already present, why are you adding the same scenario? If you wanna change something please change the step only.

Reply:
fixed